.github/workflows/verify_e2e-windows.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     env:
       CI: true
-      NODE_OPTIONS: --max-old-space-size=8192
+      NODE_OPTIONS: ${{ matrix.node-version == '20.x' && '--max-old-space-size=4096 --no-node-snapshot' || '--max-old-space-size=4096' }}
```

Comment:
I'd keep the same value, not decreasing it.

Reply:
Restored to 8192 in E2E tests